spec/math/statistics/vector_spec.rb

```diff
@@ -6,12 +6,15 @@
       before do
         @dv = Daru::Vector.new [323, 11, 555, 666, 234, 21, 666, 343, 1, 2], dtype: dtype
         @dv_with_nils = Daru::Vector.new [323, 11, 555, nil, 666, 234, 21, 666, 343, nil, 1, 2]
+        @dv_with_missing = Daru::Vector.new [1, 2, 3, 3], missing_values: [3], dtype: :gsl
+        @dv_with_all_missing = Daru::Vector.new [3, 3], missing_values: [3], dtype: :gsl
       end
```

Comment:
Had to replace all `nil`s so that the same tests can work with `:gsl`. What do you suggest? Should I test the array wrapper separately to include `nil`?